pkgs/applications/science/physics/crystfel/default.nix

```
@@ -168,6 +176,31 @@ stdenv.mkDerivation rec {
     sed -i -e 's#execlp("mosflm"#execl("${mosflm}/bin/mosflm"#' libcrystfel/src/indexers/mosflm.c;
   '';
 
+  postInstall = let
+    executables = [
+      "indexamajig"
+      "partialator"
+      "get_hkl"
+      "list_events"
+      "cell_tool"
+      "compare_hkl"
+      "cell_explorer"
+      "make_pixelmap"
+      "whirligig"
+      "ambigator"
+      "partial_sim"
+      "pattern_sim"
+      "render_hkl"
+      "process_hkl"
+      "check_hkl"
+    ] ++ lib.optionals withGui [ "crystfel" ];
+  in if withBitshuffle then
+    lib.intersperse "\n" (map (exe:
+      "wrapProgram $out/bin/${exe} --set HDF5_PLUGIN_PATH ${hdf5-external-filter-plugins}/lib/plugins")
+      executables)
+  else
+    "";
```

```suggestion
  postInstall = let
    executables = [
      "indexamajig"
      "partialator"
      "get_hkl"
      "list_events"
      "cell_tool"
      "compare_hkl"
      "cell_explorer"
      "make_pixelmap"
      "whirligig"
      "ambigator"
      "partial_sim"
      "pattern_sim"
      "render_hkl"
      "process_hkl"
      "check_hkl"
    ] ++ lib.optionals withGui [ "crystfel" ];
  in lib.optionalString withBitshuffle
    lib.intersperse "\n" (map (exe:
      "wrapProgram $out/bin/${exe} --set HDF5_PLUGIN_PATH ${hdf5-external-filter-plugins}/lib/plugins")
      executables);
```
We can't replace this with ``for file in $out/bin/ ; do ... ; done``?